libphidget22/CMakeLists.txt

```diff
@@ -39,5 +39,8 @@ install(
 )
 
 ament_environment_hooks(env_hook/libphidget22_library_path.sh)
+set(ENV_VAR_NAME "LD_LIBRARY_PATH")
+set(ENV_VAR_VALUE "opt/libphidget22/lib")
```

Comment:
I haven't tested this, but isn't there a `"/"` missing before `opt`?

Reply:
> I haven't tested this, but isn't there a `"/"` missing before `opt`?

No, we don't actually install libphidget22 to `/opt`, we install it to a relative `opt` directory.  In the case of the debian packages, that ends up being `/opt/ros/<rosdistro>/opt/libphidget22/lib`.  In the case of a local build, that ends up being `ws/install/libphidget22/opt/libphidget22/lib`.  By making this a relative path, it should work in both circumstances.

Reply:
Ok, thanks for the clarification!